batch/scripts/get_arguments.sh

```diff
@@ -66,6 +66,11 @@ do
       shift # past argument
       shift # past value
       ;;
+      -fn|--field_names)
```

Comment:
there is already an argument `--field_map` which is a json object with field name and type. Can we reuse it for this purpose?

Reply:
Thinking about this, it might be best to do a `head -1` on the source file to get the column list. We don't rename or drop column, do this should be savest

Reply:
https://github.com/wri/gfw-data-api/blob/c851ba8225da95875080990b99e14d7b9ef0cf7a/batch/scripts/create_tabular_schema.sh#L19

Reply:
https://github.com/wri/gfw-data-api/blob/c851ba8225da95875080990b99e14d7b9ef0cf7a/batch/scripts/create_tabular_schema.sh#L19